src/test/runTests.ts

```diff
@@ -0,0 +1,75 @@
+import * as path from 'path';
+import * as cp from 'child_process';
+
+const { downloadAndUnzipVSCode, runTests } = require('vscode-test');
+
+export function resolveCliPathFromVSCodeExecutablePath(vscodeExecutablePath: string) {
+	if (process.platform === 'win32') {
+		if (vscodeExecutablePath.endsWith('Code - Insiders.exe')) {
+			return path.resolve(vscodeExecutablePath, '../bin/code-insiders.cmd');
+		} else {
+			return path.resolve(vscodeExecutablePath, '../bin/code.cmd');
+		}
+	} else if (process.platform === 'darwin') {
+		return path.resolve(vscodeExecutablePath, '../../../Contents/Resources/app/bin/code');
+	} else if (process.platform === 'linux') {
+		if (vscodeExecutablePath.endsWith('code-insiders')) {
+			return path.resolve(vscodeExecutablePath, '../code-insiders');
+		} else {
+			return path.resolve(vscodeExecutablePath, '../code');
+		}
+	} else {
+		if (vscodeExecutablePath.endsWith('code-insiders')) {
+			return path.resolve(vscodeExecutablePath, '../bin/code-insiders');
+		} else {
+			return path.resolve(vscodeExecutablePath, '../bin/code');
+		}
+	}
+}
+
+async function doTheThing(vscodePath : string ) : Promise<string> {
+	return new Promise<string>( async (resolve, reject) => {
+		/**
+		 * Install Kubernetes extension
+		 */
+        const extensionId : string = 'ms-kubernetes-tools.vscode-kubernetes-tools';
+		const cliPath : string = resolveCliPathFromVSCodeExecutablePath(vscodePath);
+		console.log(`cliPath = ${cliPath}`);
+		const spawnMe = cp.spawn(cliPath, ['--install-extension', '--force', extensionId]);
```

Comment:
the command is incorrect.
here it is tryign to install the extension with id `--force` and opening the file `ms-kubernetes-tools.vscode-kubernetes-tools`

Reply:
Yup. I corrected it.